### DISCUSSION TITLE
Can't paste reliably into the admin search bar

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#4022] [Q&A]
I think there is an issue pasting content which has additional line feeds at the end.

Example:

Click the copy button to copy the following to the clipboard. Paste it into the admin search bar and it will work fine.

```
foo
```

Now copy this one. For me, this will not paste and the Paste function is disabled when cursor focus is in the search bar. The same clipboard contents will paste fine in a New Record field or in the Chrome address bar.

```
foo

```
edit: actually, `Edit > Paste` is enabled but will not paste. `Right-click > Paste` is disabled. Super odd!

**Comment by ganigeorgiev** [comment]
Hm, indeed it is not working, I'm not sure exactly why.
Maybe it is a config options of CodeMirror (the editor plugin we use for the syntax highlight and autocompletion) or I messed up something with the change events. I'll investigate it sometime later today.

**Reply by ganigeorgiev** [reply]
It seems to be a "feature" I implemented and forgot about https://github.com/pocketbase/pocketbase/blob/1208edec92905bb661d9d57f3c9407b69a4559ba/ui/src/components/base/CodeEditor.svelte#L252.
But it is definitely not intuitive and will consider updating it.

**Reply by benallfree** [reply]
`content.split(/\n/).join(' ')` maybe

**Reply by ganigeorgiev** [reply]
Should be fixed in `master` (_it ended up a little more complicated as CodeMirror supports also multiple-selections via CTRL+D; in the future I'll consider replacing eventually the editor as it is too complex and heavy for what we do_).
The fix will be available with the next minor v0.20.2 (I'll not rush publishing the release as it shouldn't be a blocker).
